ci/build.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.15.3
```

Comment:
Shouldn't we be using 1.16.2 now? (and 1.16 as a whole)

Reply:
I'll bump versions.